pkg/bootstrapper/bootstrapper.go

```diff
@@ -66,6 +66,32 @@ const (
 	// identify the nodes managed by WSU and future operators. (We could have gotten this from boostrap kubeconfig too
 	// however the label value is resolved on the host side, making it convenient when we run WMCB within a container)
 	nodeLabel = "node.openshift.io/os_id=Windows"
+
+	// kubeletExeKey is the map key used for the kubelet.exe in the deconstructed kubelet map
+	kubeletExeKey = "kubeletexe"
+	// kubeletStandAloneArgsKey is the map key used for standalone kubelet args like --windows-service in the
+	//deconstructed map
+	kubeletStandAloneArgsKey = "standalone"
+
+	// CNI constants
+	// cniDirName is the directory within the install dir where the CNI binaries are placed
+	cniDirName = "cni"
+	// cniConfigDirName is the directory in the CNI dir where the cni.conf is placed
+	cniConfigDirName = cniDirName + "/config/"
```

Comment:
FWIW, this doesn't necessarily be configured this way. While giving kubelet args, we can specify any directory that has `cni.conf` file.